.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,13 @@
+name: shellcheck-action-test
```

Comment:
`shellcheck.yml` and `cppcheck.yml` could be combined into one file with two different tests.